Redis flaky test

<!--
Thanks for submitting a bug report. Please fill the template below,
otherwise we will not be able to process this bug report.
-->

#### Summary
<!-- Summarize the problem in a few sentences: -->

Test fails in `pkg/redis`

#### Steps to Reproduce
<!-- How can we reproduce the problem? -->

1. Run tests
2. Be unlucky

<!--
Please upload relevant configuration (as .txt).
If you use the command "ttn-lw-stack config", you can redact sensitive config.
-->

#### What do you see now?
<!--
Please paste terminal output, upload logs (as .txt) or upload screenshots.
-->

```
--- FAIL: TestProtoDeduplicator (0.00s)
    doc.go:66: 
        /home/travis/gopath/src/go.thethings.network/lorawan-stack/pkg/redis/redis_test.go:778
        Expected '32.768s' to be less than or equal to '32.767s' (but it wasn't)!
FAIL
FAIL	go.thethings.network/lorawan-stack/pkg/redis	0.129s
```

#### What do you want to see instead?
<!-- Please add some examples or mock-ups if applicable. -->

Not fail

#### Environment
<!--
Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
-->

Travis

#### How do you propose to implement this?
<!-- Please think about how this could be fixed. -->

fix test

#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

No

--- a/pkg/redis/redis_test.go
+++ b/pkg/redis/redis_test.go
@@ -756,7 +756,7 @@ func TestProtoDeduplicator(t *testing.T) {
 		Encoding.EncodeToString([]byte("proto3")),
 	})
 	a.So(lockTTL, should.BeLessThanOrEqualTo, ttl)
-	a.So(listTTL, should.BeLessThanOrEqualTo, lockTTL)
+	a.So(listTTL, should.BeLessThanOrEqualTo, ttl)
 
 	ss, err = cl.LRange(DeduplicationListKey(key2), 0, -1).Result()
 	if !a.So(err, should.BeNil) {
@@ -775,5 +775,5 @@ func TestProtoDeduplicator(t *testing.T) {
 		Encoding.EncodeToString([]byte("proto2")),
 	})
 	a.So(lockTTL, should.BeLessThanOrEqualTo, ttl)
-	a.So(listTTL, should.BeLessThanOrEqualTo, lockTTL)
+	a.So(listTTL, should.BeLessThanOrEqualTo, ttl)
 }

